external/src/main/resources/application.properties

```diff
@@ -130,3 +130,24 @@ quarkus.log.cloudwatch.access-key-secret=placeholder
 # This is a new feature introduced in Quarkus 1.13 and used by the dev UI.
 # It is causing failures when the app is started with quarkus:dev because some parts of our code are not compliant with the CDI spec.
 quarkus.arc.dev-mode.monitoring-enabled=false
+
+# If true, the following property will enable policies history storage into the database defined below.
+engine.policies-history.enabled=false
+
+# The policies history service is enabled during tests.
+%test.engine.policies-history.enabled=true
+# During tests only, the required SQL schema is created using Flyway.
+%test.quarkus.flyway.migrate-at-start=true
+
+# The following database is used to store the policies history data which is then used from the policies-ui-backend project.
+# The database schema is managed by policies-ui-backend.
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/postgres
+quarkus.datasource.username=postgres
+quarkus.datasource.password=postgres
+
+# Transforms camel case entities attributes names into snake case.
+quarkus.hibernate-orm.physical-naming-strategy=com.redhat.cloud.policies.engine.history.SnakeCasePhysicalNamingStrategy
```

Comment:
Could this be an annotation? Or maybe these don't work here and that's the reason of the lack of `@Column` annotation in fields?

Reply:
`SnakeCasePhysicalNamingStrategy` is my preferred way of converting Java fields names into DB fields names because it is global, but I didn't use it in `policies-ui-backend` because that project already had some persistence code based on Panache, so I used `@Column` annotations instead to avoid introducing a regression. On the other hand, `policies-engine` didn't have any database-related code so I added `SnakeCasePhysicalNamingStrategy`.